lib/controllers/ActuatorController.h

```diff
@@ -30,16 +30,16 @@ class ActuatorController {
   mbed_error_status_t setControlMode(t_actuatorControlMode controlMode);
 
   mbed_error_status_t setMotorPower_Percentage(float percentage);
-  mbed_error_status_t setVelocity_DegreesPerSec(float degreesPerSec);
-  mbed_error_status_t setAngle_Degrees(float degrees);
+  mbed_error_status_t setVelocity_DegreesPerSec(float degreesPerSec);  // Need to mod for Servo
```

Comment:
Can you elaborate what needs to be done? Are you planning to do it in this PR? 

See other comment about making the motor and coninuous servo function calls the same.

Reply:
Hmmm, I think these are also leftover comments from earlier version...with the change we have in servo motor now, we don't really need to modify these functions for servos...
One thing to note though is that we shouldn't configure an actuator controller for a servomotor to velocity control mode...because it would be pointless. I'm not sure if we should limit this in code, or just add a comment in ServoMotor class. I don't see a good way to limit this in code, without reverting back to type checking

Reply:
continuous servos (at least cheap ones) don't really do closed loop velocity control, so it wouldn't be pointless... though not the best choice either.

Reply:
What is the best choice then? just curious

Reply:
probably just a normal motor. I guess the advantage of those kinds of servos are that they integrate a motor driver and gearbox into a compact package.